### PR TITLE
Implement: Feature to show popup with alert to re login on token expire(#2ftb1v8)

### PR DESCRIPTION
**Pull request by shashwatbangar**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/store/modules/user/UserState.ts [hunk 1]
```diff
@@ -3,4 +3,5 @@ export default interface UserState {
     current: object | null;
     selectedBrand: string;
     instanceUrl: string;
+    expirationTime: number;
```
Comment (adityasharma7) [inline src/store/modules/user/UserState.ts:6]:
```suggestion
    tokenExpirationTime: number;
```

File: src/views/orders.vue [hunk 2]
```diff
@@ -277,8 +278,20 @@ export default defineComponent({
...
     }),
   },
+  created() {
+    const currentTime = new Date().getTime();
+    if(this.expirationTime <= currentTime) {
+      this.isTokenExpired = true;
+    }
+  },
+  mounted() {
+    if(this.isTokenExpired) {
```
Comment (adityasharma7) [inline src/views/orders.vue:291]: I think this will work only for orders page. We need a generic solution that works across the app

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/store/modules/user/UserState.ts (hunk 1 above)
Comment (shashwatbangar) [reply src/store/modules/user/UserState.ts:6]: Done sir.

File: src/views/orders.vue (hunk 2 above)
Comment (shashwatbangar) [reply src/views/orders.vue:291]: Sure sir.

**Comment by ymaheshwari1** [comment]
Closing as not needed, as we have SSO in place.

**Pull request closed without merging**